.travisAndroidComponents.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Read the BUILD_TOOL_VERSION and COMPILE_SDK_VERSION
+BUILD_TOOLS_VERSION=$(cat build.gradle | grep "BUILD_TOOLS_VERSION*" | sed -e "s/^.*'\(.*\)'/\1/")
+COMPILE_SDK_VERSION=$(cat build.gradle | grep "COMPILE_SDK_VERSION*" | sed -e "s/^.*= \(.*\)$/\1/")
```

Comment:
can we put this in the `.travis.yml`, too?

Reply:
No. Unfortunately this isn't possible because of the limitations of the travis.yml file